tdvt/TDVT Results.twb

```diff
@@ -59,17 +60,16 @@
       <connection class='federated'>
         <named-connections>
           <named-connection caption='test_results_combined' name='textscan.1gc4mna00d9z6u19vsqbk09t94y2leaf'>
-            <connection class='textscan' directory='/Users/tsjoblad/Desktop/new TDVT Workbook/For GitHub (1:11:21)' filename='test_results_combined.csv' password='' server='' workgroup-auth-mode='as-is' />
+            <connection class='textscan' directory='/Users/s.davis/Desktop' filename='test_results_combined.csv' password='' server='' workgroup-auth-mode='as-is' />
           </named-connection>
         </named-connections>
         <_.fcp.ObjectModelEncapsulateLegacy.false...relation connection='textscan.1gc4mna00d9z6u19vsqbk09t94y2leaf' name='test_metadata.csv' table='[test_metadata#csv]' type='table'>
-          <columns character-set='UTF-8' header='yes' locale='en_US' separator='&#9;'>
-            <column datatype='string' name='Test Function' ordinal='0' />

```

Comment:
Is it intentional that we no longer have Test Function column from test_metadata in the twb?

Reply:
I didn't remove it. It was not in the original. I did however add the Function as a filter...

Reply:
@rosswbrown It looks like the `Test Function` col was removed from the csv file in 6154c338cfabcf72a312b04f5c68d2067046ad24 - this must be the first time refreshing the workbook since.